### PR TITLE
[Tutorials]Fix tutorial examples

### PR DESCRIPTION
**Pull request by zsluedem**
## Motivation
I was just going through the tutorials and found some errors.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/move-language/move/blob/main/CONTRIBUTING.md#developer-workflow)?

Yes

## Test Plan

Only documents are changed. No test plans

### CODE REVIEW

File: language/documentation/tutorial/README.md [hunk 1]
```diff
@@ -592,7 +592,7 @@ Informally speaking, the block `spec balance_of {...}` contains the property spe
 
 Let's first run the prover using the following command inside [`BasicCoin` directory](./step_7/BasicCoin/):
 ```bash
-move prove
+move package prove
```
[the remark this answers is not in the document]
Comment (zsluedem) [reply language/documentation/tutorial/README.md]: Oh. Sorry. I installed the old version of `move` and I thought the right command is move package prove.
Thanks!

**Review by tnowacki — APPROVED** [review]
Thanks!

**Comment by zsluedem** [comment]
I tried the ci locally and it passed. Is it possible to run the ci again?

**Pull request merged**